docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: <your title>
+permalink: /docs/rfcs/<file-name>
+---
+
+* RFC PR: [<PR #>](https://github.com/mozilla-mobile/android-components/pull/<#>)
+* Start date: YYYY-MM-DD (day of posting)
+* End date: YYYY-MM-DD (initial deadline for feedback, subject to blocking changes required)
```

Comment:
Do we keep the PRs open until the end date even when approved for others to have the ability to comment?

Reply:
Updated in a couple places to make clear that proposals can be merged early once they've been approved